build.sbt

```diff
@@ -139,12 +139,23 @@ lazy val node = (project in file("node"))
     maintainer in Linux := "Pyrofex, Inc. <info@pyrofex.net>",
     packageSummary in Linux := "RChain Node",
     packageDescription in Linux := "RChain Node - the RChain blockchain node server software.",
+     linuxPackageMappings += {
```

Comment:
Indentation is off here.  Also please make sure that your editor is using tabs instead of spaces.